label/data/report_paperformat.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+  <data>
+    <record id="paperformat_label" model="report.paperformat">
+      <field name="name">Label A4</field>
+      <field name="format">custom</field>
+      <field name="default" eval="True" />
+      <field name="page_height">297</field>
+      <field name="page_width">210</field>
+      <field name="orientation">Portrait</field>
+      <field name="margin_top">0</field>
+      <field name="margin_bottom">0</field>
+      <field name="margin_left">0</field>
+      <field name="margin_right">0</field>
+      <field name="header_line" eval="False" />
+      <field name="header_spacing">0</field>
+      <field name="dpi">144</field>
+    </record>
+    
+    <report id="dynamic_label" 
+	    string="Qweb Label" 
```

Comment:
Adding new code is a good opportunity to auto-indent all xmls correctly, with 4 spaces for tags and aligned attributes, e.g. there is 1 space missing here.